src/stats/overall.ts

```diff
@@ -187,8 +186,7 @@ function generateInputs(game: SlippiGame): Array<PlayerInput> {
 
     // Do the same for c-stick
     const prevCstickRegion = getJoystickRegion(prevPlayerFrame.cStickX, prevPlayerFrame.cStickY);
-    // FIXME: stop using any. it is not clear what type this is. cstickX does not exist on the type PreFrameUpdateType
-    const currentCstickRegion = getJoystickRegion((playerFrame as any).cStickX, (playerFrame as any).cStickY);
```

Comment:
Pretty sure these comments are wrong because I do see those fields in PreFrameUpdateType